backend/core/entities/docker_entity.py

```
@@ -0,0 +1,330 @@
+"""Docker domain entities"""
+
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional, Any
+from datetime import datetime
+from enum import Enum
+from ..value_objects.host_connection import HostConnection
```

Ruff (0.11.9)

`..value_objects.host_connection.HostConnection` imported but unused

Remove unused import: `..value_objects.host_connection.HostConnection`

(F401)